src/components/GlobalSidebar/index.js

```diff
@@ -64,7 +64,8 @@ class GlobalSidebar extends Component {
           {children}
           <li>
             <a tabIndex="-1" className="pointer" onClick={() => this.setState({ settingsModalOpen: true }, handleOpen(false))}>
-              <i className="ss-icon ss-settings vertical-align-middle" />{' '}Settings
+              <i className="ss-icon ss-settings vertical-align-middle" />{' '}
+              <LocaleFormattedMessage id="nav.settings" defaultMessage="Settings" />
```

Comment:
we can use `setting.title` for this, no need to duplicate keys with same values.